amf-cli/shared/src/test/scala/amf/emit/CompatibilityCycleTest.scala

```diff
@@ -22,11 +22,25 @@ class CompatibilityCycleTest extends AsyncFunSuite with FileAssertionTest {
   private val basePath = "amf-cli/shared/src/test/resources/compatibility/"
 
   test("RAML with union of nil + type to OAS 3.0") {
-    compatibility("raml10/union-nil-plus-element.raml", "oas30/union-nil-plus-element.yaml", Raml10YamlHint, Oas30YamlHint)
+    compatibility(
+      "raml10/union-nil-plus-element.raml",
+      "oas30/union-nil-plus-element.yaml",
+      Raml10YamlHint,
+      Oas30YamlHint
+    )
   }
 
   test("RAML with union of nil + multiple types to OAS 3.0") {
-    compatibility("raml10/union-nil-plus-elements.raml", "oas30/union-nil-plus-elements.yaml", Raml10YamlHint, Oas30YamlHint)
+    compatibility(
+      "raml10/union-nil-plus-elements.raml",
+      "oas30/union-nil-plus-elements.yaml",
+      Raml10YamlHint,
+      Oas30YamlHint
+    )
+  }
+
+  test("OAS 3.0 with nullable parameter to JSON") {
```

Comment:
Why would we run the compatibility pipeline for this? Compatibility pipeline is for Spec conversion, not syntax conversion.

Reply:
Because this comes from a CRI in which a client uses APID to transfrom from yaml to json, going through the compatibility pipeline, thus encountering this error.

Other discussion if this should be the correct pipeline, but tried other pipelines and got the same result if that helps.

Reply:
Hmm this isn't convincing, as the use case per se is wrong. I don't think it is wise to have tests for unsupported use cases. If the customer wants to go from YAML to JSON then they shouldn't use compatibility pipeline.

Reply:
The problem with this is that if another member of the team makes a change that breaks this test, they will not know that this is an unsupported/incorrect use case and therefore will try to maintain it, ending up in having to maintain something that we don't want to.

Reply:
we discussed it internally, it would be better if we add this as an "editing" case instead of a "compatibility" case.

Reply:
after discussing it internally, its ok to me.